modules/cli/src/main/scala/scala/cli/commands/InstallHome.scala

```diff
@@ -0,0 +1,67 @@
+package scala.cli.commands
+
+import caseapp._
+import coursier.env.{EnvironmentUpdate, ProfileUpdater}
+import scala.io.StdIn.readLine
+
+import scala.util.Properties
+
+object InstallHome extends ScalaCommand[InstallHomeOptions] {
+  override def hidden: Boolean = true
+  def run(options: InstallHomeOptions, args: RemainingArgs): Unit = {
+
+    val binDirPath      = scala.build.Directories.default().binRepoDir
+    val scalaCliBinPath = binDirPath / "scala-cli"
+
+    if (os.exists(scalaCliBinPath)) {
+      if (options.force) os.remove.all(scalaCliBinPath)
+      else if (coursier.paths.Util.useAnsiOutput()) {
```

Comment:
What if the session is interactive? I guess Util.useAnsiOutput() covers it, but I'm not sure

Reply:
It should. It doesn't have the right name, it's not really about ANSI output. Its value is computed [here](https://github.com/coursier/coursier/blob/f9246587f62020afa152f67dfd14927d57eae02d/modules/paths/src/main/java/coursier/paths/Util.java#L121-L146). It returns false if `System.console() == null` in particular, or on CIs.